docker/puppet-runtime/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM centos:8
```

Comment:
This didn't fix Travis :(

Reply:
Will add in Buildkit on Travis in a separate PR.... though this container is so simple it's probably not a big deal either way.